dot/sync/chain_sync_integeration_test.go

```diff
@@ -332,7 +337,7 @@ func TestWorkerToRequests(t *testing.T) {
 					StartingBlock: *variadic.MustNewUint64OrHash(1),
 					EndBlockHash:  nil,
 					Direction:     network.Ascending,
-					Max:           &max128,
+					Max:           &max9,
```

Comment:
i think this was updated, can you pull development again?

Reply:
I've fixed this to match #2256.